data_steward/analytics/cdr_ops/fitbit_qc.py

```diff
@@ -149,7 +149,7 @@
 FROM
   `{{project}}.{{dataset}}.{{table_name}}` t
 JOIN
-  `{{project}}.{{sandbox_dataset}}._deactivated_participants` d
+  `{{project}}.{{dataset}}_sandbox._deactivated_participants` d
```

Comment:
I would suggest keeping the whole `sandbox_dataset` parameter and assigning it in the format statement.